### PR TITLE
fix(@ciscospark/i-p-conversation): allow markdown when creating 1:1

### PR DESCRIPTION
**Pull request by aimeex3**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes posting markdown when creating a 1:1 space. Previously, it only took `displayName`. However, it needs to handle `content` for markdown.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] send a markdown message when creating a 1:1 space

**Test Configuration**:
* SDK Version
* Node/Browser Version: 8.9.4
* NPM Version: 5.6.0

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ const Conversation = SparkPlugin.extend({
    * @param {Array<Participant>} params.participants
    * @param {Array<File>} params.files
    * @param {string} params.comment
+   * @param {string} params.markdown
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js]: The sdk doesn't explicitly handle markdown; let's call this `html` to make it clear that it should already have been formatted before it hits `Conversation#create()`.

File: packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js [hunk 2]
```diff
@@ -873,8 +874,8 @@ const Conversation = SparkPlugin.extend({
       user: params.participants[1]
     }), Object.assign(options, {includeConvWithDeletedUserUUID: true, includeParticipants: true}))
       .then((conversation) => {
-        if (params.comment) {
-          return this.post(conversation, {displayName: params.comment})
+        if (params.comment || params.markdown) {
+          return this.post(conversation, {displayName: params.comment, content: params.markdown})
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js]: I think this'll fail eslint.

File: packages/node_modules/@ciscospark/internal-plugin-conversation/test/integration/spec/create.js [hunk 3]
```diff
@@ -163,6 +179,13 @@ describe('plugin-conversation', function () {
       .then((c) => spark.internal.conversation.get(c, {activitiesLimit: 2}))
       .then((c) => assert.equal(c.activities.items[1].object.displayName, 'comment')));
 
+    it('creates a conversation with a markdown comment', () => spark.internal.conversation.create({comment: 'comment', markdown: '<strong>comment</strong>', participants})
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/test/integration/spec/create.js]: Similarly, let's use "richtext" or "html" in the test descriptions. Markdown is handled at a higher layer.

File: packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js (hunk 2 above)
Comment (aimeex3) [reply packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js]: hmm i'm not seeing any errors for this line in eslint. what should it be?

**Pull request merged**